test/recipe/admin/utils/giveMe.js

```diff
@@ -52,6 +52,17 @@ export default {
     return { key: keyAndTestId, title, component };
   },
 
+  anErrorPage({ safeLinks, errorInfos, data }){
+    return {
+      type: "ErrorPage",
+      schema: { safeLinks, errorInfos },
+      data: {
+        type: "Inline",
+        value: data
+      }
+    };
+  },
```

Comment:
error infos should be constructed using another stubber, like we do in `giveMe.aHeaderItem`